test/cypress/integration/my-services/update-service-name.cy.js

```diff
@@ -26,34 +26,30 @@ function setupStubs (serviceName, stubs = []) {
 
 describe('Update service name', () => {
   beforeEach(() => {
-    // keep the same session for entire describe block
-    Cypress.Cookies.preserveOnce('session')
+    cy.setEncryptedCookies(authenticatedUserId)
   })
 
   describe('Edit a service name without a Welsh name', () => {
-    it('should display the my services page', () => {
-      cy.setEncryptedCookies(authenticatedUserId)
-      setupStubs(serviceName)
+    it('should allow updating service name', () => {
```

Comment:
Can we get rid of the function defined on line 19 (setupStubs)?

Reply:
It's still used in 2 places, but I guess we could inline it?

Reply:
That's fine, I didn't realise it is still being used.
I will approve this PR - as this PR is to make them standalone.